src/main/java/com/iota/iri/controllers/AddressViewModel.java

```diff
@@ -38,6 +42,30 @@ private AddressViewModel(Address hashes, Indexable hash) {
         this.hash = hash;
     }
 
+    /**
+     * Constructor for an {@link Address} set controller from an existing {@link Address} set.
+     * The resulting hash list is sorted
```

Comment:
This doc is unclear to me. This is not a java constructor...

Maybe write:
"Loads all transaction that mutate a certain address. It sorts them by the attachment timestamp"